content/oci/readonlyoci.go

```diff
@@ -186,6 +186,25 @@ func loadIndex(ctx context.Context, index *ocispec.Index, fetcher content.Fetche
 	return nil
 }
 
+// loadIndex loads index into memory.
+func loadIndexWithMemoryWithDelete(ctx context.Context, index *ocispec.Index, fetcher content.Fetcher, tagger content.Tagger, graph *graph.MemoryWithDelete) error {
```

Comment:
This should be defined in `deletableOci.go` as it has nothing to do with `readonlyoci.go`.

Reply:
Good point, thanks.